observIQ/bindplane-op
# Use npm ci for dev target and trust make to build when needed

### Proposed Change
Add ui/node_modules target that depends on `ui/package.json` and `ui/package-lock.json` and will only do an npm ci when necessary.

To test:

First run should npm ci
```
make dev
```

Second run will skip npm ci
```
make dev
```

Simulate a package-lock.json change and it will npm ci
```
touch ui/package-lock.json
make dev
```



##### Checklist
